src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Ball getRandomColor() {
+        int ballNumber = random.nextInt(1, 101);

```

Comment:
let's make 101 a MAX_BOUND constant, don't forget we use CONSTANT_CASE to name them